programs/uxd/src/events.rs

```diff
@@ -96,6 +96,19 @@ pub struct SetDepositoryMintingDisabledEvent {
     pub minting_disabled: bool,
 }
 
+/// Event called in [instructions::edit_*_depository::handler].
+#[event]
+pub struct SetDepositoryProfitsBeneficiaryKeyEvent {
+    #[index]
+    pub version: u8,
+    #[index]
+    pub controller: Pubkey,
+    #[index]
+    pub depository: Pubkey,
+    #[index]
+    pub profits_beneficiary_key: Pubkey,
```

Comment:
I wouldn't call that key as we don't use that nomenclature anywhere else

Reply:
yes, left the same comments on the crefix PR, was for alignment for the naming,
but gonna get rid of this field now but just `profits_beneficiary_collateral` as discussed